packages/block-editor/src/components/writing-flow/index.js

```diff
@@ -89,6 +90,16 @@ class WritingFlow extends Component {
 		 * @type {?DOMRect}
 		 */
 		this.verticalRect = null;
+
+		this.appender = createRef();
+	}
+
+	componentDidMount() {
+		window.addEventListener( 'mousedown', this.switchToEditMode );
```

Comment:
Might be something we want to use [`withGlobalEvents`](https://github.com/WordPress/gutenberg/tree/master/packages/compose/src/higher-order/with-global-events), but I'm starting to question its usefulness, particularly with React hooks making this simple.

Reply:
We could have a `useGlobalEvent` or something like that. I guess the benefit would be to only have a unique event handler?